Assets/Scripts/WorldEngine/Manager.cs

```diff
@@ -386,11 +322,17 @@ private Manager()
 
         _lastUpdatedCells = new HashSet<TerrainCell>();
 
-        /// static initalizations
+        // static initializations
 
         Tribe.GenerateTribeNounVariations();
     }
 
+    /// <summary>Determines if the required control and shift keys were used.</summary>
```

Comment:
This function also checks that user is not interacting with an Input Field in the UI before allowing the key shortcut to proceed